submission-history/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
We should also ignore `.vs`. After updating and committing `.gitignore`, you'll need to untrack the directory with `git rm --cached -r .vs` and commit the changes.

Reply:
Added two gitignore files